drivers/hmis/app/models/hmis/form/processor.rb

```diff
@@ -0,0 +1,196 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Form::Processor < ::GrdaWarehouseBase
+  self.table_name = :hmis_assessment_processors
+
+  has_one :assessment_detail
+
+  # assessment is accessed through the assessment_detail
+  belongs_to :health_and_dv, class_name: 'Hmis::Hud::HealthAndDv', primary_key: :HealthAndDVID, optional: true
+  belongs_to :income_benefit, class_name: 'Hmis::Hud::IncomeBenefit', primary_key: :IncomeBenefitID, optional: true
+  belongs_to :enrollment_coc, class_name: 'Hmis::Hud::EnrollmentCoc', primary_key: :EnrollmentCocID, optional: true
+  belongs_to :physical_disability, class_name: 'Hmis::Hud::Disability', primary_key: :DisabilityID, optional: true
+  belongs_to :developmental_disability, class_name: 'Hmis::Hud::Disability', primary_key: :DisabilitiesID, optional: true
+  belongs_to :chronic_health_condition, class_name: 'Hmis::Hud::Disability', primary_key: :DisabilitiesID, optional: true
+  belongs_to :hiv_aids, class_name: 'Hmis::Hud::Disability', primary_key: :DisabilitiesID, optional: true
+  belongs_to :mental_health_disorder, class_name: 'Hmis::Hud::Disability', primary_key: :DisabilitiesID, optional: true
+  belongs_to :substance_use_disorder, class_name: 'Hmis::Hud::Disability', primary_key: :DisabiliesID, optional: true
```

Comment:
These are going to need a `data_source_id` if we're joining on the HUD keys.  Could we just switch to using their internal IDs?

Reply:
Good question -- I forgot we had Rails IDs...